templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -458,6 +452,12 @@ Parameters:
     AllowedValues: [General Purpose (SSD), Provisioned IOPS]
     ConstraintDescription: Must be 'General Purpose (SSD)' or 'Provisioned IOPS'.
     Type: String
+  InternetFacingLoadBalancer:
+    Default: true
```

Comment:
Should this be false by default?

Reply:
I'm not sure 🤔 Maybe? Really depends on how most of our customers us our products

Reply:
This was already true by default. So this isn't a change.